github3/repos/repo.py

```diff
@@ -1033,6 +1057,18 @@ def delete(self):
         return self._boolean(self._delete(self._api), 204, 404)
 
     @requires_auth
+    def delete_branch(self, name):
+        """Delete a branch. Returns False if branch doesn't exist
+        or not authorized.
+
+        :param str branch: (required), the branch to delete.
+
+        :returns: bool -- True if successful, False otherwise
+        """
+        url = self._build_url('git/refs/heads', name, base_url=self._api)
```

Comment:
Isn't there already a delete method for the reference that's created from `create_ref`?